### PR TITLE
New package: kstars-2.9.6

### PR DESCRIPTION
**Pull request by travankor**

### CODE REVIEW

File: srcpkgs/kstars/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'kstars'
+pkgname=kstars
+version=2.9.6
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="extra-cmake-modules"
```
Comment (Chocimier) [inline srcpkgs/kstars/template]: Please add `pkg-config`.

**Comment by maxice8** [comment]
musl fails because of:
```
/builddir/kstars-2.9.6/kstars/htmesh/SkipListElement.h:26:17: error: 'LONG_LONG_MAX' was not declared in this scope
 #define KEY_MAX LONG_LONG_MAX
```

**Comment by travankor** [comment]
Closing since I got `stellarium` to work. Feel free to use this if musl gets fixed.

**Comment by abenson** [comment]
How'd you get stellarium working?

**Comment by travankor** [comment]
In `~/.stellarium/config.ini`,

```
[plugins_load_at_startup]
...
Satellites                             = false
...
```

I think there is some other Void-specific bug (one of the QT patches?), but this works for now.

**Pull request closed without merging**